### PR TITLE
docs: updated 'podman-mac-helper' documentation verification examples

### LINKED ISSUE #3640: podman-macos-helper documentaion is outdated
### Bug description

Current version on pdman-macos-helper is printing out help that does not have commands mentioned on documentation page

https://github.com/containers/podman-desktop/blob/f38537de95c945599d3bb6bcb54e7ffb0834120a/website/docs/migrating-from-docker/using-podman-mac-helper.md?plain=1#L45

It references `run` command that is not mentioned in help output.

https://github.com/containers/podman-desktop/blob/f38537de95c945599d3bb6bcb54e7ffb0834120a/website/docs/migrating-from-docker/using-podman-mac-helper.md?plain=1#L51

```sudo podman-mac-helper --help
podman-mac-helper is a system helper service and tool for managing docker.sock

Usage:
  podman-mac-helper [command]

Available Commands:
  help        Help about any command
  install     installs the podman helper agent
  uninstall   uninstalls the podman helper agent

Flags:
  -h, --help   help for podman-mac-helper

Use "podman-mac-helper [command] --help" for more information about a command.
```

### Operating system

macOS Ventura 13.4 (Apple M1)

### Installation Method

Installer from website/GitHub releases

### Version

1.3.1

### Steps to reproduce

_No response_

### Relevant log output

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by themr0c**
### What does this PR do?

docs: updated 'podman-mac-helper' documentation verification examples

### What issues does this PR fix or reference?

fixes #3640

### CODE REVIEW

File: website/docs/migrating-from-docker/using-podman-mac-helper.md [hunk 1]
```diff
@@ -25,32 +25,31 @@ The service redirects `/var/run/docker` to the fixed user-assigned UNIX socket l
...
 
 #### Verification
 
-1. Your tools communicating to the Docker socket, such as [Maven](https://maven.apache.org/) or [Testcontainers](https://www.testcontainers.org/), communicate with Podman without reconfiguration.
+1. The Docker socket is a symbolic link for the Podman socket:
 
-2. Use the `podman-mac-helper` tool to run commands.
-   To run a command with Podman by using the `podman-mac-helper` tool, prefix the command with `podman-mac-helper`.
+   ```shell-session
+   $ ls -la /var/run/docker.sock
```
Comment (benoitf) [inline website/docs/migrating-from-docker/using-podman-mac-helper.md:41]: I suppose it should say that symbolic link is pointing to a podman.sock file

File: website/docs/migrating-from-docker/using-podman-mac-helper.md [hunk 2]
```diff
@@ -25,34 +25,57 @@ The service redirects `/var/run/docker` to the fixed user-assigned UNIX socket l
...
 
    ```shell-session
-   $ podman-mac-helper run -it <your_container> bash
+   $ DOCKER_HOST=unix:///var/run/docker.sock docker info --format=json | jq -r .ServerVersion
+   ```
+
+1. (Optionally, if the `docker` CLI is installed) The docker CLI context is set to the default value `unix:///var/run/docker.sock`:
+
+   ```shell-session
+   $ docker context
```
Comment (benoitf) [inline website/docs/migrating-from-docker/using-podman-mac-helper.md]:
```suggestion
   $ docker context list
```

it's missing the list keyword else it displays the help of context command

File: website/docs/migrating-from-docker/using-podman-mac-helper.md [hunk 3]
```diff
@@ -25,34 +25,57 @@ The service redirects `/var/run/docker` to the fixed user-assigned UNIX socket l
...
-2. Use the `podman-mac-helper` tool to run commands.
-   To run a command with Podman by using the `podman-mac-helper` tool, prefix the command with `podman-mac-helper`.
+1. (Optionally, if the `docker` CLI is installed) The `docker` CLI communicates with the Podman socket.
 
-   Example:
+   Therefore this command outputs Podman version rather that Docker version:
 
    ```shell-session
-   $ podman-mac-helper run -it <your_container> bash
+   $ DOCKER_HOST=unix:///var/run/docker.sock docker info --format=json | jq -r .ServerVersion
```
Comment (benoitf) [inline website/docs/migrating-from-docker/using-podman-mac-helper.md]: maybe remove ` DOCKER_HOST=unix:///var/run/docker.sock ` as if context is correctly set it means that docker is using this host already

Comment (themr0c) [reply website/docs/migrating-from-docker/using-podman-mac-helper.md]: And then switch with the next step.

**Review by benoitf — APPROVED** [review]

**Comment by themr0c** [comment]
@benoitf thank you for helping me get through this.

**Pull request merged**